### PR TITLE
fix: orgs list international bug

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What type of PR is this?

- [ ] Feature
- [x] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes # orgs list international bug

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/app/common/utils/str-num-date.tsx [hunk 1]
```diff
@@ -14,8 +14,15 @@
 import i18n, { getCurrentLocale } from 'i18n';
 import { isArray } from 'lodash';
 import moment, { Moment } from 'moment';
+import 'moment/locale/zh-cn';
```
Comment (McDaddy) [inline shell/app/common/utils/str-num-date.tsx:17]: is this import required?

Comment (hujiahao-hjh) [reply shell/app/common/utils/str-num-date.tsx:17]: The internationalization of the moment here does not take effect without import this.

Comment (daskyrk) [reply shell/app/common/utils/str-num-date.tsx:17]: This is imported in core, not take effect?

Comment (hujiahao-hjh) [reply shell/app/common/utils/str-num-date.tsx:17]: Yes, this will only take effect if import it again here.

**Pull request merged**